jest.component.setup.js

```diff
@@ -1,5 +1,5 @@
 const oldEnv = Object.assign({}, process.env);
-
+jest.setTimeout(20000);
```

Comment:
Code smell. Why this?

Reply:
comopnent tests take a bit time to run since we start a process call it and wait for files to be written

Reply:
Cannot you make a wait for the file to appear with a timeout? At least one would know what's going on

Reply:
Please make something that works like

```
await fileCreated('<path>', <timeoutInS>)
```